### DISCUSSION TITLE
WebSocket Server?

### DISCUSSION THREAD

**Discussion by rhapsodyv** [bluenviron/mediamtx#1139] [General]
Is there any plans to stream rtsp to a websocket video player?

**Comment by aler9** [comment]
Hello, i considered WebSocket for a while, but then i choose to focus on Low-Latency HLS, since it's more standard and more supported (WebSocket video players depend on MSE extensions, that are not available on iPhone). Probably WebSocket has a slightly lower delay (anyway, not lower than 1sec), we'll see about it in the future.

EDIT: after some testing, i can say that Low-latency HLS is slightly faster than WebSocket/MSE, therefore, there's no reason to implement WebSocket/MSE.

**Comment by aler9** [comment]
In order to decrease latency, probably it's better to switch directly to WebRTC.

[a comment by an automated account is not shown]
